torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -27,6 +27,25 @@ void CheckGraphEligibility(const std::shared_ptr<torch::jit::Graph>& graph) {
       throw std::runtime_error("Cannot accelerate unfrozen graphs");
     }
   }
+  // We support output types include None, Tensor and List or Tuple of Tensor
```

Comment:
you probably want to give a bit friendlier error message, at least print the output type and say what it's allowed to be